src/paperless_tesseract/parsers.py

```diff
@@ -45,13 +45,22 @@ def get_thumbnail(self):
         """
 
         out_path = os.path.join(self.tempdir, "convert.png")
+        gs_out_path = os.path.join(self.tempdir, "gs_out.png")
 
         # Run convert to get a decent thumbnail
+
+        # https://github.com/danielquinn/paperless/issues/447
+        # call gs first
+        cmd = ["gs", "-q", "-sDEVICE=pngalpha",
+               "-o", gs_out_path, self.document_path]
+        if not subprocess.Popen(cmd).wait() == 0:
+            raise ParseError("Thumbnail (gs) failed at {}".format(cmd))
+        # then run convert on the output from gs
         run_convert(
             self.CONVERT,
             "-scale", "500x5000",
             "-alpha", "remove",
-            "{}[0]".format(self.document_path),
+            "gs_out_path",
```

Comment:
I think quoting this is wrong and will not work, i.e. it should be:

```suggestion
            gs_out_path,
```

Reply:
Dumb mistake.. thanks.
Interestingly enough, it still worked when I tested it..